lib/fontist/utils/git_requirement.rb

```diff
@@ -0,0 +1,21 @@
+module Fontist
```

Comment:
Can we use https://github.com/ruby-git/ruby-git here?

Reply:
Let's see :)

Reply:
Done. Kept the GitRequirement class for more human-readable error if there is no git in a system.